drivers/client_access_control/app/controllers/client_access_control/history_controller.rb

```diff
@@ -54,134 +54,20 @@ def queue
     end
 
     def pdf
-      @user = User.setup_system_user
-      current_user ||= @user # rubocop:disable Lint/UselessAssignment
+      @client_history = ClientHistory.new(client_id: params[:client_id].to_i, user_id: params[:user_id].to_i, years: params[:years]&.to_i)
+      @client ||= @client_history.client
```

Comment:
Don't need this any more right?
```suggestion
```

Reply:
Oh, that's so the test will pass.  It's fine, but I wonder why it's needed for the test?

Reply:
Yeah, this is fine.  Maybe a quick comment above?
```
# set @client for when this method is called outside of the controller context.
```

Reply:
just pushed it up!